application/templates/home.html

```diff
@@ -1,11 +1,22 @@
-<html>
+<!doctype html>
+<html lang="en">
 	<head>
+		
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		
+		<script src="{{ url_for('static', filename='jquery-3.4.1.js') }}"></script>

```

Comment:
Don't include your own copy of jquery like this. Get it from a cdn like you are doing with the bootstrap css/js files.